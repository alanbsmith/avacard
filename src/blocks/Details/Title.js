import { gridScale } from '../../utils';

import Header from '../../elements/Header';

const Title = Header.extend`
  padding: ${gridScale(2)} 0;
`;

export default Title;
