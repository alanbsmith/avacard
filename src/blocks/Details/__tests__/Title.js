import { generateSnapshots } from "__tests__/helpers";

import Title from '../Title';

generateSnapshots(Title);
