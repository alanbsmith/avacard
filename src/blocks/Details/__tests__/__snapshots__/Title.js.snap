// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`styled.h2 renders correctly 1`] = `
.c0 {
  color: rgba(41,71,98,0.8);
  font-size: 20px;
  font-weight: 300;
  line-height: 1.5em;
  -webkit-letter-spacing: 0;
  -moz-letter-spacing: 0;
  -ms-letter-spacing: 0;
  letter-spacing: 0;
  margin: 0;
  text-transform: capitalize;
  padding: 1.25rem 0;
}

<h2
  className="c0"
/>
`;
